File: AmazonClient/AmazonClient/AlwaysIgnoreCertPolicy.vb
Imports System.Net
Imports System.Security.Cryptography.X509Certificates

Public Class AlwaysIgnoreCertPolicy
    Implements ICertificatePolicy
    Public Function CheckValidationResult(ByVal srvPoint As ServicePoint, _
                                          ByVal cert As X509Certificate, ByVal request As WebRequest, _
                                          ByVal certificateProblem As Integer) As Boolean Implements ICertificatePolicy.CheckValidationResult
        ' return TRUE to force the certificate to be accepted.
        Return True
    End Function
End Class

File: WMInventoryGoodsMovements/WMInventoryGoodsMovements/AlwaysIgnoreCertPolicy.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Net;
using System.Security.Cryptography.X509Certificates;


namespace WMInventoryGoodsMovements
{
    class AlwaysIgnoreCertPolicy : ICertificatePolicy
    {
        public bool CheckValidationResult(ServicePoint srvPoint, X509Certificate cert, WebRequest request, int certificateProblem)
        {
            // return TRUE to force the certificate to be accepted.
            return true;
        }
    }
}
